packages/zosfiles/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to the Zowe z/OS files SDK package will be documented in thi
 
 ## Recent Changes
 
+- Enhancement: When copying partitioned data sets that have identical member names, the user is now prompted to confirm before the operation occurs in case of potential data loss. [#2349] (https://github.com/zowe/zowe-cli/issues/2349)
```

Comment:
See above comment regarding labeling as a bug fix. We could rephrase this entry to justify the change:

```suggestion
- BugFix: Users were not warned when copying partitioned data sets with identical member names. Now, the user is prompted to confirm before continuing the copy operation to avoid potential data loss. [#2349] (https://github.com/zowe/zowe-cli/issues/2349)
```